Integrations/integration-RSANetWitness_v11.1.yml

```diff
@@ -83,7 +83,7 @@ script:
                 'https': https
             }
             return proxies
-        return None
+        return {"http": None, "https": None}
```

Comment:
we should also delete proxy environment vars in this case

Reply:
if you really want you can use this http://docs.python-requests.org/en/master/api/#requests.Session.trust_env

but it's an overkill.. this explicitly tells requests not to use a proxy

Reply:
I meant:
```
    del os.environ['HTTP_PROXY']
    del os.environ['HTTPS_PROXY']
    del os.environ['http_proxy']
    del os.environ['https_proxy']
```

Reply:
but why?